gcloud-java-compute/src/main/java/com/google/gcloud/compute/ComputeImpl.java

```diff
@@ -914,6 +933,99 @@ public Operation delete(final AddressId addressId, OperationOption... options) {
     }
   }
 
+  @Override
+  public Operation create(SnapshotInfo snapshot, final OperationOption... options) {
```

Comment:
Nit: `OperationOption...` doesn't have to be final, right?  (Is our general policy to avoid using final for method parameters when unnecessary?)